doc/source/flexibility/maximum_binlog_size.rst

```diff
@@ -0,0 +1,57 @@
+.. _maximum_binlog_size:
+
+========================================
+ Restricting the number of binlog files
+========================================
+
+Estimating the number of binary logs and disk space use is a challenge. |Percona Server| has the variable :variable:`binlog_space_limit`, which sets the maximum amount of space to use for all binary logs. With the :variable:`binlog_space_limit` set to `0`, the default value, the variable is turned off and no limit is set. You set the :variable:`binlog_space_limit` on and provide a size limit. The value specified is not a hard limit. A single transaction is written to a single binary log, and the size of this transaction may exceed the limit.
```

Comment:
* Not only a single transaction: as the documentation later describes, the last binlog file will never be pruned.
* Please use consistent grammar, e.g. With variable set..., you set, ...

Reply:
rewritten.